_1_basics/src/main/java/code/_4_student_effort/FizzBuzz.java

```diff
@@ -0,0 +1,45 @@
+package code._4_student_effort;
+
+public class FizzBuzz {
+    public static void main(String args[]) {
+
+        for(int i = 1; i<=100; i++){
```

Comment:
pentru numarul 105, care este divizibil cu 3,5,7 ar trebui sa afisezi FizzBuzzRizz

cate contitii noi si if-uri trebuie sa adaugi pt asta ?

Reply:
Salut! Ar trebui sa mai am un if statement inaintea tuturor if-urilor din for. Acel if o sa aiba 3 conditii(i%3/5/7 == 0). Eu asa as implementa, daca ar fi fost si 105 inclus in for.